sdk/common/src/main/java/io/opentelemetry/sdk/common/InstrumentationScopeInfo.java

```diff
@@ -26,15 +27,30 @@ public abstract class InstrumentationScopeInfo {
 
   /** Creates a new instance of {@link InstrumentationScopeInfo}. */
   public static InstrumentationScopeInfo create(String name) {
-    requireNonNull(name, "name");
-    return new AutoValue_InstrumentationScopeInfo(name, null, null);
+    return InstrumentationScopeInfo.create(name, null, null, Attributes.empty());
   }
 
   /** Creates a new instance of {@link InstrumentationScopeInfo}. */
   public static InstrumentationScopeInfo create(
```

Comment:
I'll deprecate this in a followup PR. Doing it touches 50+ files so better keep that as a dedicated refactor.